piccolo_api/crud/endpoints.py

```diff
@@ -199,14 +200,19 @@ def pydantic_model_optional(self) -> t.Type[pydantic.BaseModel]:
             model_name=f"{self.table.__name__}Optional",
         )
 
-    def pydantic_model_plural(self, include_readable=False):
+    def pydantic_model_plural(
+        self,
+        include_readable=False,
+        exclude_columns: t.Tuple[Column, ...] = (),
+    ):
         """
         This is for when we want to serialise many copies of the model.
         """
-        base_model = create_pydantic_model(
+        base_model: t.Any = create_pydantic_model(
             self.table,
             include_default_columns=True,
             include_readable=include_readable,
+            exclude_columns=exclude_columns,
```

Comment:
This makes me realise that `create_pydantic_model` should probably have an `include_columns` argument.

Reply:
That's a good idea. This could be useful for some things.